mllib/src/main/scala/org/apache/spark/ml/regression/GeneralizedLinearRegression.scala

```diff
@@ -1152,4 +1170,32 @@ class GeneralizedLinearRegressionTrainingSummary private[regression] (
         "No p-value available for this GeneralizedLinearRegressionModel")
     }
   }
+
+  /**
+   * Summary table with feature name, coefficient, standard error,
+   * tValue and pValue.
+   */
+  @Since("2.2.0")
+  lazy val summaryTable: DataFrame = {
+    if (isNormalSolver) {
+      var featureNames = featureName
+      var coefficients = model.coefficients.toArray
+      var idx = Array.range(0, coefficients.length)
+      if (model.getFitIntercept) {
+        featureNames = featureNames :+ "Intercept"
+        coefficients = coefficients :+ model.intercept
+        // Reorder so that intercept comes first
+        idx = (coefficients.length - 1) +: idx
+      }
+      val result = for (i <- idx.toSeq) yield
+        (featureNames(i), coefficients(i), coefficientStandardErrors(i), tValues(i), pValues(i))
+
+      val spark = SparkSession.builder().getOrCreate()
+      import spark.implicits._
```

Comment:
I was using the spark session and implicits to be able to use `toDF` to create data frame with names from `Seq`. Could you explain how this `import dataset.sparkSession.implicits._` works? Could not import it in spark shell.
```
<console>:56: error: not found: value dataset
       import dataset.sparkSession.implicits._
```